.github/workflows/sync-config-schema.yaml

```diff
@@ -114,9 +129,11 @@ jobs:
 
 
       - name: Update vcluster schema in vcluster-config
-        if: ${{ steps.release.outputs.update_main == 'false' }}
+        # update only on beta, -rc and stable versions
+        if: ${{ steps.release.outputs.is_alpha_version == 'false' }}
```

Comment:
`steps.release.outputs.is_stable_version == 'false' && && steps.release.outputs.is_alpha_version == 'false' `?

Reply:
no, we want to do this on betas, RC and full versions (see my comment below)